app/src/androidTest/java/org/mozilla/fenix/ui/util/Strings.kt

```diff
@@ -11,3 +11,4 @@ const val FRENCH_LANGUAGE_HEADER = "Langues"
 const val FR_SETTINGS = "Paramètres"
 const val FRENCH_SYSTEM_LOCALE_OPTION = "Utiliser la langue de l’appareil"
 const val ROMANIAN_LANGUAGE_HEADER = "Limbă"
+const val ARABIC_LANGUAGE_HEADER = "اللغة"
```

Comment:
Mind grouping the header strings together?

Reply:
Done !